cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc

```diff
@@ -129,6 +129,12 @@ Tests for the absence of a pattern predicate.
 | label:yes[]
 |
 
+| <<query-plan-directed-all-relationships-scan,DirectedAllRelationshipsScan>>
```

Comment:
I stole this from https://github.com/neo4j/neo4j-documentation/pull/1507, it used to be the case that this was not a necessary step but happened automatically somehow but I guess not you have to do it manually?